### ISSUE TITLE
"Delete all gaps in all seqs" does not delete terminal gap characters in shorters eqs

### ISSUE THREAD

**Issue by Shellfishgene** [AliView/AliView#56]
I'm not sure if this is intentional: When deleting all gaps in all sequences, gaps at staggered ends are not delete, for example
```
ATG---
ATGAC-
```
becomes
```
ATG--
ATGAC
```
Makes sense as it's not really a gap, but does not work for removing all gap characters for subsequent programs.

**Comment by andersla** [comment]
Yes this is a side effect because AliView is automatically padding the end of sequences untill all are equal length when saving.
If not padding sequences to same length there are other programs not being able to open alignment.
We have to think about alternatives to solve this use-case when you dont want sequences to be padded.

**Comment by andersla** [comment]
I think the best option is not to pad sequences automatically - but to ask the user if they want them padded if they are not when saving.
